src/zjs_callbacks.c

```diff
@@ -95,16 +96,23 @@ static int zjs_ringbuf_error_max = 0;
 static int zjs_ringbuf_last_error = 0;
 
 #ifdef DEBUG_BUILD
-static void set_info_string(char **str, const char *file, const char *func)
+static void set_info_string(char *str, const char *file, const char *func)
 {
-    uint32_t file_len = strlen(file);
+    char *i = (char*)(file + strlen(file));
```

Comment:
Wait, actually this should be const char *i on the left and you shouldn't need a cast on the right.